core/src/components/segment-button/segment-button.tsx

```diff
@@ -36,10 +36,15 @@ export class SegmentButton implements ComponentInterface, ButtonInterface {
 
   @State() checked = false;
 
+  /**
+   * The `id` of the segment content.
+   */
+  @Prop({ reflect: true }) contentId?: string;
+
   /**
    * If `true`, the user cannot interact with the segment button.
    */
-  @Prop({ mutable: true }) disabled = false;
+  @Prop({ mutable: true, reflect: true }) disabled = false;
```

Comment:
Not sure why this wasn't reflected before, but I can revert if we don't want this. It was really just to make testing the behavior of buttons not being allowed to be disabled with a segment view easier

Reply:
I don't see a reason for reflecting it if it works fine without it.

Reply:
Reverted :)